### DISCUSSION TITLE
Timeline for Toggle component

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1592] [Q&A] [answered]
Hello friends, 
Is there a timeline for the Paste Toggle component? 
Thanks!

**Comment by TheSisb** [comment, marked as answer]
Hi @sang-yeo,

According to our roadmap, it should be expected around Q4 2021.  It's under the `2021.11.16` release on our [roadmap page](https://paste.twilio.design/roadmap).  Keep in mind that the further out we are from the release time-wise, the less reliable the specific dates become.  

Hope this helps!

**Reply by sang-yeo** [reply]
Thank you!
